src/en/components/button/base.md

```diff
@@ -6,7 +6,8 @@ permalink: false
 tags: ['buttonEN', 'header']
 ---
 
-# Button <br>&lt;gcds-button&gt;
+<h1 class="mb-0">Button</h1>
```

Comment:
I don't know if this is the best way to do a subtitle for the `h1`. I have seen some places online recommend using a `header` like 
```
<header>
  <h1>Button</h1>
  <code>gcds-button</code>
</header>
```
Another option could be to keep the `code` tag in the `h1` and just provide it additional style to wrap to the new line.